avm/res/service-networking/traffic-controller/main.bicep

```
@@ -207,21 +226,41 @@ output associations array = [
   }
 ]
 
+@description('The security policies of the Application Gateway for Containers.')
+output securityPolicies array = [
```

Hey @krbar,
if you want to make it discoverable, you can add:

```suggestion
output securityPolicies {
  @description('The name of the security policy.')
  name: string
  @description('The resource ID of the security policy.')
  resourceId: string
} = [
```